cmd/dockerd/daemon.go

```diff
@@ -113,6 +113,7 @@ func (cli *DaemonCli) start(opts *daemonOptions) (err error) {
 	}
 
 	system.InitLCOW(cli.Config.Experimental)
+	system.InitContainerdOnWindows(cli.Config.Experimental, cli.Config.ContainerdAddr)
```

Comment:
Does this have to be executed at this point, or could it be at the same point where Linux does this initialisation? Initialisation of systemd on Linux is some lines lower (`if cli.Config.ContainerdAddr == "" && runtime.GOOS != "windows"`).

Perhaps extract to a function, and have different implementations in a `_windows.go` and `_linux.go` file?

Reply:
Yes, that's addressed now.